src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,7 +1,23 @@
 import React from 'react';
 
 import './UserInfo.scss';
+import { User } from '../../react-app-env';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  currentUser: User | undefined,
```

Comment:
You shouldn't render this component if user is undefined

Reply:
Just fix type

Reply:
Done! Thank you!